addons/event/data/event_demo.xml

```diff
@@ -5,27 +5,6 @@
              <field name="groups_id" eval="[(4, ref('event.group_event_user'))]"/>
      </record>
 
-    <!-- Demo Data for Event Type -->
```

Comment:
@tde-banana-odoo,
I moved some event type demo data in event_sale module, because as per our spec when create event type then create an automatic product of the event type.
And removed <code>event_type_id</code> from <code>product.template</code> and from everywhere removed code related to <code>event_type_id</code>.